llvm/lib/Target/DirectX/DXILShaderFlags.cpp

```diff
@@ -51,7 +51,10 @@ void ComputedShaderFlags::print(raw_ostream &OS) const {
   if (FlagVal == 0)
     return;
   OS << "; Note: shader requires additional functionality:\n";
-#define SHADER_FEATURE_FLAG(bit, FlagName, Str)                                \
+#define SHADER_FEATURE_FLAG(bit, DxilModuleNum, FlagName, Str)                 \
```

Comment:
Your suggestion is good, but also please fix the capitalization on the first argument:

```suggestion
#define SHADER_FEATURE_FLAG(Bit, DxilModuleBit, FlagName, Str)                 \
```

Reply:
Fixed.